### DISCUSSION TITLE
Aqara Pet Food Feeder Icon

### DISCUSSION THREAD

**Discussion by z-master42** [elax46/custom-brand-icons#708] [Q&A]
Why has the icon for the Aqara Pet Food Feeder `phu:aqara-pet-feeder-c1` been removed?
Has the icon for the fountain `phu:pet-water-fountian` been added instead?

BTW: I have just realised that there is also a spelling mistake.

**Comment by rchiileea** [comment]
@elax46 I just spotted this as well, the pet feeder is missing, I think issues  keep happening when others have commited icons to the repo, they are either oversized or icons go missing

**Comment by elax46** [comment]
Hi, thanks for reporting, I don't know why but these problems are occurring even though I'm keeping PR to a minimum
